tests/test_sudoku.py

```diff
@@ -1,6 +1,6 @@
 from sudoku import Sudoku
 from cell import Cell
-from resources import EASY_TASK, EASY_SOLUTION, HARD_TASK, HARD_SOLUTION
+from resources import EASY_TASK, EASY_SOLUTION, HARD_TASK, HARD_SOLUTION, HARD_TASK_2, HARD_TASK_2_SOLUTION, HARD_TASK_3, HARD_TASK_3_SOLUTION
```

Comment:
Непоследовательное именование
```suggestion
from resources import EASY_TASK, EASY_SOLUTION, HARD_TASK, HARD_SOLUTION, HARD_TASK_2, HARD_SOLUTION_2, HARD_TASK_3, HARD_SOLUTION_3
```